components/config-model-migrator/README.md

```diff
@@ -0,0 +1,59 @@
+# New Config Model Migration Tool
+
+This tool will help to identify and migrate the old xml configurations to the new configuration model 
+and create the deployment.toml file, which suites the new config model that comes with
+ Identity Server 5.9.0 or above.
+
+* It will create the `deployement.toml` file with the existing knowledge that identified before and 
+committed in the `Catalog.csv` file hosted in resources folder of this tool.
+
+* The rest of the newly identified changes will be recorded in a new `outputCatalog.csv` 
+file inside the output folder.
+
+* The not-templated, changed files will be filtered to a folder called `unTemplatedFiles` inside the 
+output folder.
+
+* The logs will be recorded in the `log.txt` file inside the output folder.
+
+## How to run the tool
+
+1. Build the `identity-tools` repository and get the `org.wso2.configuration.migrater-<version>-bundle.zip` 
```

Comment:
Build instructions should be more descriptive

- Cloning/Downloading the tool
- Build command
- Where to find the zip